src/components/NewForm/QuestionsList.js

```diff
@@ -84,24 +84,58 @@ const QuestionList = (props) => {
     }
   };
 
-  const questionAction = (index) => {
-    return (
-      <div title="edit">
-        <EditFilled onClick={() => handleQuestionEdit(index)} />
-      </div>
+  const handleQuestionClone = (index) => {
```

Comment:
Lots of redundancy here, we should think of ways to reduce this once all usability features are merged